### PR TITLE
Feature/rst 479 features

### PR DESCRIPTION
**Pull request by zaparka**

### CODE REVIEW

File: spec/presenters/guide_presenter_spec.rb [hunk 1]
```diff
@@ -2,25 +2,32 @@
...
+        to yield_successive_args(['some_text', 'some_content'], ['and_more_text', 'some_content'])
+    end
+
     it "passes the files basename and rendered content to the block argument" do
       allow(File).to receive(:read).and_return mock_content
 
       expect(renderer).to receive(:render).with mock_content
-      expect { |probe| subject.each_rendered_file &probe }.
-        to yield_successive_args(%w<some_text some_content>, %w<and_more_text some_content>)
+      guide_presenter.each_rendered_file { |name| name }
```
Comment (leoapost) [inline spec/presenters/guide_presenter_spec.rb]: Why this line is there? Is it testing something?

File: spec/features/admin_panel/admin_index_feature_spec.rb [hunk 2]
```diff
@@ -2,19 +2,24 @@
 
 RSpec.feature 'Viewing the admin interfaces index page', type: :feature do
 
-  let!(:out_of_filter_range_claim) do
+  let(:out_of_filter_range_claim) do
     create :claim,
-      submitted_at: Date.new(2015, 06, 01),
-      created_at: Date.new(2015, 06, 01)
+      submitted_at: Date.new(2015, 0o6, 0o1),
```
[the remark this answers is not in the document]
Comment (zaparka) [reply spec/features/admin_panel/admin_index_feature_spec.rb]: So I run rubocop autocorrect and looks it doesn't like 06, so he made an octal from it. Will fix that.

[the remark this answers is not in the document]
Comment (leoapost) [reply spec/features/admin_panel/admin_index_feature_spec.rb]: Yeah, I had the feeling that Rubocop is behind that. Could you please make sure it's fixed in every file?

File: spec/presenters/guide_presenter_spec.rb (hunk 1 above)
Comment (zaparka) [reply spec/presenters/guide_presenter_spec.rb]: @leoapost it's triggering the test.

Comment (zaparka) [reply spec/presenters/guide_presenter_spec.rb]: @leoapost changes submitted

**Comment by zaparka** [comment]
@leoapost green build :)

**Pull request merged**